### ISSUE TITLE
failed to check proof: should have 2 roots

### ISSUE THREAD

**Issue by akme** [xssnick/tonutils-go#175]
Some litenodes were updated in testnet yesterday and now we get this error on GetBlockShardsInfo():
v.1.8.9
```
failed to check proof: should have 2 roots
```
v1.9.0 return timeout

Could you please take a look?

**Comment by akme** [comment]
UPD:
v1.9.0 error
```
failed to parse message: failed to parse field Data of adnl.MessageAnswer, err: failed to parse struct from bytes for interface {}, err: failed to parse field Proof of ton.AllShardsInfo, err: incorrect cells len in field []*cell.Cell
```

**Comment by xssnick** [comment]
Hi, new liteserver version is not sends shard state proof in `GetBlockShardsInfo` in second root of proof anymore. 

For now you can import lib like that `go get -u github.com/xssnick/tonutils-go@41495ebfc0d81266892021ed9ba7ba3d01766c7d`, so it will work in both new testnet and old mainnet.

A bit later i will release 1.9.1

**Comment by xssnick** [comment]
Released [v1.9.1](https://github.com/xssnick/tonutils-go/releases/tag/v1.9.1)

**Comment by akme** [comment]
Thanks! All good now!
